.buildkite/hooks/pre-command

```diff
@@ -65,18 +65,18 @@ export PLUGIN_PORTAL_SECRET
 vault read -field=key secret/release/signing >$KEY_FILE
 KEYPASS_SECRET=$(vault read -field=passphrase secret/release/signing)
 export KEYPASS_SECRET
-export KEY_ID_SECRET=D88E42B4
```

Comment:
What's the reason?

Reply:
Gradle needed to know the ID of the key to use during the signing process as mentioned [here](https://docs.gradle.org/current/userguide/signing_plugin.html#sec:signatory_credentials). That's why I had to set it up. However, now we've changed to the "[in-memory](https://docs.gradle.org/current/userguide/signing_plugin.html#sec:in-memory-keys)" approach, which doesn't need that value. So since it's no longer needed, I removed it.